packages/astro/src/core/build/generate.ts

```diff
@@ -335,7 +335,7 @@ async function getPathsForRoute(
 			logger,
 			ssr: serverLike,
 		}).catch((err) => {
-			logger.debug('build', `├── ${bold(red('✗'))} ${route.component}`);
+			logger.info(null, `├── ${bold(red('✗'))} ${route.component}`);
```

Comment:
Makes sense to me. I feel like we could also make this more specific though, maybe something like:
```suggestion
			logger.error('build, `Failed to call getStaticPaths for ${route.component}`);
```